src/ResourceManagement/BatchAI/BatchAIJobImpl.cs

```diff
@@ -126,6 +161,30 @@ public ToolTypeSettings.PyTorch.Definition.IBlank<BatchAIJob.Definition.IWithCre
             return new PyTorchImpl(new PyTorchSettings(), this);
         }
 
+        public ContainerImageSettings.Definition.IBlank<BatchAIJob.Definition.IWithCreate> DefineContainerSettings(string image)
```

Comment:
>ContainerImageSettings.Definition.IBlank<BatchAIJob.Definition.IWithCreate> [](start = 15, length = 75)

these all methods should return Impls. If you get errors during compilation you should check if your Imls correctly implement all the interfaces.